changelog.md

```diff
@@ -1,5 +1,24 @@
 # CHANGELOG.md
 
+## 0.3.11 (2019-12-19)
+
+Breaking changes:
+
+  - Rename `fake()` to `generate()`
+  - Rename `unmock.newFaker()` to `unmock.faker()`
```

Comment:
These methods were added after the previous release so `fake` and `unmock.newFaker()` have not been part of any release, so nothing breaking 😇

Reply:
Should I move them to the Features section then or rather remove them entirely?

Reply:
I think the one about `unmock.faker()` could be added to features after `UnmockFaker`, the other one can be removed. But it's not that important, not sure if anyone reads change logs anyway 🙂